README.md

```diff
@@ -1,16 +1,10 @@
-# ![nfcore/test-datasets](docs/images/test-datasets_logo.png)
+# test-datasets: `atacseq`
 Test data to be used for automated testing with the nf-core pipelines
```

Comment:
Could you add a comment for which pipeline (link?!) this is? It should be completely self-explanatory per branch what this contains and for which pipeline :-) Thank you :-)